AnkiDroid/src/main/java/com/ichi2/audio/AudioRecordingController.kt

```diff
@@ -186,10 +188,22 @@ class AudioRecordingController :
         setUpMediaPlayer()
 
         audioTimer = AudioTimer(this, this)
-        recordButton.setOnClickListener {
-            Timber.i("primary 'record' button clicked")
-            controlAudioRecorder()
-        }
+
+        // if the recorder is in the 'cleared' state
+        // holding the 'record' button should start a recording
+        // releasing the 'record' button should complete the recording
+        // TODO: remove haptics from the long press - the 'buzz' can be heard on the recording
+        recordButton.setOnHoldListener(object : OnHoldListener {
+            override fun onTouchStart() {
+                Timber.d("pressed 'record' button'")
+                controlAudioRecorder()
+            }
+
+            override fun onHoldEnd() {
+                Timber.d("finished holding 'record' button'")
+                controlAudioRecorder()
+            }
```

Comment:
This one's on me - messed up in the proposed patch, could you test, then this is good to go

```suggestion
             override fun onHoldEnd() {
                 Timber.d("finished holding 'record' button'")
                 if (state is ImmediatePlayback) {
                     controlAudioRecorder()
                 } else {
                     // if we're recording audio to add permanently, 
                     // releasing the button after a long press should be 'save', not 'pause'
                     saveButton?.performClick()
                 }
             }
```